### DISCUSSION TITLE
Error: Command "npm run build" exited with 1

### DISCUSSION THREAD

**Discussion by CarlitoDevDreamer** [vercel/vercel#6515] [Help]
Area: Vercel CLI
Name: arduinoprojectportugal
Previous Page: Not Specified
-----------------------------------
21:08:31.746 Retrieving list of deployment files...
21:08:33.927 Downloading 10 deployment files...
21:08:34.049 Analyzing source code...
21:08:35.558 Installing build runtime...
21:08:38.317 Build runtime installed: 2.759s
21:08:41.187 Looking up build cache...
21:08:41.944 Build cache found. Downloading...
21:08:44.295 Build cache downloaded [25.80 MB]: 2351.317ms
21:08:45.452 Detected package.json
21:08:45.452 Installing dependencies...
21:08:49.290 npm WARN read-shrinkwrap This version of npm is compatible with lockfileVersion@1, but package-lock.json was generated for lockfileVersion@2. I'll try to do my best with it!
21:08:56.158 npm WARN @testing-library/user-event@12.8.3 requires a peer of @testing-library/dom@>=7.21.4 but none is installed. You must install peer dependencies yourself.
21:08:56.168 npm WARN tsutils@3.21.0 requires a peer of typescript@>=2.8.0 || >= 3.2.0-dev || >= 3.3.0-dev || >= 3.4.0-dev || >= 3.5.0-dev || >= 3.6.0-dev || >= 3.6.0-beta || >= 3.7.0-dev || >= 3.7.0-beta but none is installed. You must install peer dependencies yourself.
21:08:56.178 npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.3.2 (node_modules/fsevents):
21:08:56.178 npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
21:08:56.187 npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules/watchpack-chokidar2/node_modules/fsevents):
21:08:56.187 npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
21:08:56.195 npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules/webpack-dev-server/node_modules/fsevents):
21:08:56.195 npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
21:08:56.197 up to date in 10.19s
21:08:57.260 145 packages are looking for funding
21:08:57.260 run `npm fund` for details
21:08:57.525 Running "npm run build"
21:08:57.797 > buda-velho@0.1.0 build /vercel/path0
21:08:57.797 > react-scripts build
21:08:59.564 Could not find a required file.
21:08:59.564 Name: index.html
21:08:59.565 Searched in: /vercel/path0/public
21:08:59.578 npm ERR! code ELIFECYCLE
21:08:59.578 npm ERR! errno 1
21:08:59.582 npm ERR! buda-velho@0.1.0 build: `react-scripts build`
21:08:59.582 npm ERR! Exit status 1
21:08:59.582 npm ERR!
21:08:59.582 npm ERR! Failed at the buda-velho@0.1.0 build script.
21:08:59.582 npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
21:08:59.600 npm ERR! A complete log of this run can be found in:
21:08:59.600 npm ERR! /vercel/.npm/_logs/2021-07-24T20_08_59_583Z-debug.log
21:08:59.617 **Error: Command "npm run build" exited with 1** 

- [ ]  ssdsd

answer from vercel:

Hi there,

Thank you for getting in touch, I'm sorry you're having difficulty using the Vercel platform.

To help us provide assistance, please provide more context on the issue being faced by answering the questions below:

    What behavior are you experiencing?
    What behavior were you expecting?
    What have you tried to resolve the issue?

Once we have answers to the above questions, we will do our best to provide you with the assistance required to resolve your issue.

Matthew Sweeney
▲ Director of Customer Success at Vercel

********************************

<html><body>
<div style="color:black;font-size:12pt;font-family:Calibri,Helvetica,sans-serif;">
I am expereincing the following behaviour, first run npx create-react-app my-app<br>

<br>

then, use cd my-app</div>
<div style="color:black;font-size:12pt;font-family:Calibri,Helvetica,sans-serif;">
<br>

</div>
<div style="color:black;font-size:12pt;font-family:Calibri,Helvetica,sans-serif;">
after       vercel<br>

<br>

And everything goes ok</div>
<div style="color:black;font-size:12pt;font-family:Calibri,Helvetica,sans-serif;">
<br>

</div>
<div style="color:black;font-size:12pt;font-family:Calibri,Helvetica,sans-serif;">
after a few changes on the code, the following errrors start appearing:<br>

<br>

<br>

21:08:31.746 | Retrieving list of deployment files...
-- | --
21:08:33.927 | Downloading 10 deployment files...
21:08:34.049 | Analyzing source code...
21:08:35.558 | Installing build runtime...
21:08:38.317 | Build runtime installed: 2.759s
21:08:41.187 | Looking up build cache...
21:08:41.944 | Build cache found. Downloading...
21:08:44.295 | Build cache downloaded [25.80 MB]: 2351.317ms
21:08:45.452 | Detected package.json
21:08:45.452 | Installing dependencies...
21:08:49.290 | npm WARN read-shrinkwrap This version of npm is  compatible with lockfileVersion@1, but package-lock.json was generated  for lockfileVersion@2. I'll try to do my best with it!
21:08:56.158 | npm WARN @testing-library/user-event@12.8.3 requires a  peer of @testing-library/dom@>=7.21.4 but none is installed. You must  install peer dependencies yourself.
21:08:56.168 | npm WARN tsutils@3.21.0 requires a peer of  typescript@>=2.8.0 \|\| >= 3.2.0-dev \|\| >= 3.3.0-dev \|\| >=  3.4.0-dev \|\| >= 3.5.0-dev \|\| >= 3.6.0-dev \|\| >= 3.6.0-beta \|\|  >= 3.7.0-dev \|\| >= 3.7.0-beta but none is installed. You must  install peer dependencies yourself.
21:08:56.178 | npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.3.2 (node_modules/fsevents):
21:08:56.178 | npm WARN notsup SKIPPING OPTIONAL DEPENDENCY:  Unsupported platform for fsevents@2.3.2: wanted  {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
21:08:56.187 | npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules/watchpack-chokidar2/node_modules/fsevents):
21:08:56.187 | npm WARN notsup SKIPPING OPTIONAL DEPENDENCY:  Unsupported platform for fsevents@1.2.13: wanted  {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
21:08:56.195 | npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules/webpack-dev-server/node_modules/fsevents):
21:08:56.195 | npm WARN notsup SKIPPING OPTIONAL DEPENDENCY:  Unsupported platform for fsevents@1.2.13: wanted  {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
21:08:56.197 | up to date in 10.19s
21:08:57.260 | 145 packages are looking for funding
21:08:57.260 | run `npm fund` for details
21:08:57.525 | Running "npm run build"
21:08:57.797 | > buda-velho@0.1.0 build /vercel/path0
21:08:57.797 | > react-scripts build
21:08:59.564 | Could not find a required file.
21:08:59.564 | Name: index.html
21:08:59.565 | Searched in: /vercel/path0/public
21:08:59.578 | npm ERR! code ELIFECYCLE
21:08:59.578 | npm ERR! errno 1
21:08:59.582 | npm ERR! buda-velho@0.1.0 build: `react-scripts build`
21:08:59.582 | npm ERR! Exit status 1
21:08:59.582 | npm ERR!
21:08:59.582 | npm ERR! Failed at the buda-velho@0.1.0 build script.
21:08:59.582 | npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
21:08:59.600 | npm ERR! A complete log of this run can be found in:
21:08:59.600 | npm ERR! /vercel/.npm/_logs/2021-07-24T20_08_59_583Z-debug.log
21:08:59.617 | Error: Command "npm run build" exited with 1

<br>

</div>
<div style="color:black;font-size:12pt;font-family:Calibri,Helvetica,sans-serif;">
<br>

</div>
<div style="color:black;font-size:12pt;font-family:Calibri,Helvetica,sans-serif;">
<br>

</div>
<div style="color:black;font-size:12pt;font-family:Calibri,Helvetica,sans-serif;">
I was expecting to see my app launched with vercel.</div>
<div style="color:black;font-size:12pt;font-family:Calibri,Helvetica,sans-serif;">
<br>

</div>
<div style="color:black;font-size:12pt;font-family:Calibri,Helvetica,sans-serif;">
<br>

</div>
<div style="color:black;font-size:12pt;font-family:Calibri,Helvetica,sans-serif;">
<br>

</div>
<div style="color:black;font-size:12pt;font-family:Calibri,Helvetica,sans-serif;">
I tried almost every thing, I uninstalled other packages, but I am not 
quite sure wich ones should I maintain and wich ones to delete. If you 
need , I can grant you access to my computer by teamviewer.</div>
<div style="color:black;font-size:12pt;font-family:Calibri,Helvetica,sans-serif;">
<br>

</div>
<div style="color:black;font-size:12pt;font-family:Calibri,Helvetica,sans-serif;">
thanks for your repply</div>
</body>
</html>

*********************************

<html><body>
<div style="color:black;font-size:12pt;font-family:Calibri,Helvetica,sans-serif;">
Also if I run this comand on the terminal: vercel version vercel CLI 23.0.1<br>

<br>

npx create-react-app my-app cd my-app vercel</div>
<div style="color:black;font-size:12pt;font-family:Calibri,Helvetica,sans-serif;">
<br>

</div>
<div style="color:black;font-size:12pt;font-family:Calibri,Helvetica,sans-serif;">
the default app runs well with the react logo rotating, linke in this this page:<br>

<br>

<a href="https://my-ppa.vercel.app/" target="_blank" rel="noopener noreferrer" data-auth="NotApplicable" id="LPlnk" data-linkindex="0">https://my-ppa.vercel.app/</a><br>

<div>
<div id="LPBorder_GTaHR0cHM6Ly9teS1wcGEudmVyY2VsLmFwcC8." style="width: 100%; max-width: 800px; margin-top: 16px; margin-bottom: 16px; min-width: 424px; position: relative;">

React App  Web site created using create-react-app  my-ppa.vercel.app
--

</div>
</div>
<br>

but if I make a simple change. I cant run the app anymore, What should I do?</div>
</body>
</html>

**Comment by RuchikaKadam** [comment]
I am facing the same issue.
